GeoIP2/project.json

```diff
@@ -2,25 +2,28 @@
   "version": "2.7.0-beta2",
   "name": "MaxMind.GeoIP2",
   "description": "MaxMind GeoIP2 Database Reader and Web Service Client",
+
   "packOptions": {
-    "authors": [
-      "MaxMind, Inc."
-    ],
-    "tags": [
-      ""
-    ],
+    "authors": ["Gregory Oschwald", "Mark Fowler"],
+    "tags": ["maxmind", "ip", "geoip", "geoip2", "geolocation", "ip2location", "iptolocation", "maxmind", "ipv4", "ipv6"],
```

Comment:
It was in another package name (under tags): https://www.nuget.org/packages/Plarium.Geo/ (not sure why this package is using `geoip2` as tag).. If we search GeoIP2 on NuGet we get three results including official package, Plarium.Geo and Plarium.Geo.Client. :(